### PR TITLE
Add third party app LiveStreamLinkGUI

### PR DESCRIPTION
**Pull request by techmouse**
I hope I did it right.

### CODE REVIEW

File: docs/applications.rst [hunk 1]
```diff
@@ -20,3 +20,18 @@ Streamlink Twitch GUI
...
+
+LiveStreamLinkGUI
+---------------------
+
+.. image:: https://raw.githubusercontent.com/techmouse/livestreamlinkgui/master/screenshot.png
+    :alt: LiveStreamLinkGUI
+
+:Description: A lightweight BASH script
+:Type: Graphical User Interface
+:OS: Linux and any other OS that supports BASH, Zenity, and typical *nix commands
```
Comment (gravyboat) [inline docs/applications.rst]: This should probably be `|MacOS| |Linux|`

**Comment by gravyboat** [comment]
I added one note. I think the main thing I forgot was the potential to rename the app/references to Streamlink (sorry I forgot to mention that when I looked at it this morning). I know @bastimeyer will bring up the same concern but I think that is the only point of note.

**Comment by gravyboat** [comment]
@techmouse Whoops missed one more thing, this should be in the third party apps file.

File: docs/applications.rst (hunk 1 above)
Comment (gravyboat) [inline docs/applications.rst]: The `*` is causing this to fail on the doc build in Travis. Should be okay if you remove that or change it to `\*`.

**Comment by techmouse** [comment]
Wait so everything I put in this file is supposed to be in a different file?

**Comment by gravyboat** [comment]
@techmouse Yeah there are two files, one is first party apps, the other is third party, you can just move it and we will squash at the end so the history is clean.

File: docs/applications.rst [hunk 2]
```diff
@@ -20,3 +20,18 @@ Streamlink Twitch GUI
...
+.. image:: https://raw.githubusercontent.com/techmouse/livestreamlinkgui/master/screenshot.png
+    :alt: LiveStreamLinkGUI
+
+:Description: A lightweight BASH script
+:Type: Graphical User Interface
+:OS: |Linux| |MacOS| And any other OS that supports BASH, Zenity, and typical \*nix commands
+:Author: `Mouse <https://github.com/techmouse>`_
+:Website: https://github.com/techmouse/livestreamlinkgui
+:Source code: https://github.com/techmouse/livestreamlinkgui
+:Info: A simple GUI for Streamlink, Livestreamer and any future forks. Easily configurable and written in BASH for low resource *nix based media machines. Designed to be compatible with streaming sites, direct links to media (both local and remote), and webpages containing direct links to media. Able to save links, open popout chats in your browser, and generally aims be an all-in-one go-to tool for watching whatever content possible in your favorite video player, instead of your browser.
```
Comment (gravyboat) [inline docs/applications.rst]: We need to escape the `*` on line 37 as well: `*nix based media machines` sorry for missing it.

**Comment by techmouse** [comment]
I'm going to be honest. This is my first time doing a pull request and I have no idea what's going on. I'm used to doing things the old school way. :P

**Comment by gravyboat** [comment]
@techmouse Don't worry about it. I'm going to close this one since the other is open.

**Pull request closed without merging**